src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ElasticPoolCrudTests.ps1

```diff
@@ -297,4 +297,57 @@ function Test-RemoveElasticPool
     {
         Remove-ResourceGroupForTest $rg
     }
+}
+
+<#
+	.SYNOPSIS
+	Test listing and cancelling a elastic pool operation
+#>
+function Test-ListAndCancelElasticPoolOperation
+{
+	# Setup
+	$location = "southeastasia";
```

Comment:
@payiAzure please avoid hard-coding locations in tests -- this potentially makes it difficult for other users to re-record this test who might not have access to the resources in the provided location. We recommend using the [`Get-Location`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L583-L604) function in our test common code to get a location that works for the user re-recording the test.